packages/indexer-api/src/routers/balances.ts

```diff
@@ -0,0 +1,13 @@
+import { Router } from "express";
+import Redis from "ioredis";
+import { BalancesController } from "../controllers/balances";
+import { BalancesService } from "../services/balances";
+
+export function getRouter(redis: Redis): Router {
+  const router = Router();
+  const service = new BalancesService(redis);
+  const controller = new BalancesController(service);
+  router.get("/hubPoolBalance", controller.getHubPoolBalance);
+  router.get("/spokePoolBalance", controller.getSpokePoolBalance);
```

Comment:
I think the REST good practice is to not use camel case in the urls
```suggestion
  router.get("/hubpool-balance", controller.getHubPoolBalance);
  router.get("/spokepool-balance", controller.getSpokePoolBalance);
```